vernac/egramml.ml

```diff
@@ -31,13 +31,21 @@ type ('self, 'tr, _, 'r) ty_rule =
 type ('self, 'r) any_ty_rule =
 | AnyTyRule : ('self, _, 'act, Loc.t -> 'r) ty_rule -> ('self, 'r) any_ty_rule
 
+let keyword_needed = function
+  | GramTerminal _ :: _ -> false
+  | GramNonTerminal _ :: _ | [] -> true (* at level 0 of the grammar, it could be "false" *)
```

Comment:
This comment is a bit cryptic.

Reply:
What was meant is "if entries at level 0 of the grammar were always delimited by a terminal on both sides, we would not need to declare the token as a keyword".

I'm not sure anymore that we want everything delimited at level 0, so I removed the comment.